packages/context/src/context.ts

```diff
@@ -246,20 +262,19 @@ export class Context {
    * the optional binding was not found.
    */
   get<T>(
-    keyWithPath: string,
+    keyWithPath: string | BindingKey<T>,
```

Comment:
Maybe it's better to define a type alias as:
```ts
export type BindingKeyType<T> = string | BindingKey<T>; 
```

Reply:
I created the type alias as `BindingAddress`.